apps/keyboard/test/build/integration/keyboard_test.js

```diff
@@ -100,10 +100,18 @@ suite('Keyboard layouts building tests', function() {
 
           return path.basename(filename, '.js');
         });
-
-      var layouts = layoutIds.map(function(layout) {
-        return 'js/layouts/' + layout + '.js';
-      });
+      
+     var layoutIds1 = [];
+     layoutIds1 = layoutIds.filter(function(layout) {
+     	if(layout)
+     		return layout;
```

Comment:
The correct array to filter is the array returned by `fs.readdirSync(appDirPath + '/js/layouts')` -- we need to use a `filter()` here and filter out file names not ended with `.js`.